guides/gatherers.md

```diff
@@ -1575,3 +1577,119 @@ Example output (in Rhai):
 // hacluster_has_default_password
 true;
 ```
+
+<span id="ini_files"></span>
+
+### ini_files@v1
+
+**Argument required**: yes.
+
+This gatherer fetches the content from a configuration file in INI format. The configuration file is specified as argument, choosen from a list of allowed files.
```

Comment:
Should it not be chosen instead of choosen ?

Reply:
Fixed in https://github.com/trento-project/wanda/pull/588/commits/f0b1114fccd734367f473eb08c69b0dd247fe174